docker-compose.yml

```diff
@@ -24,6 +24,9 @@ services:
       WATSONX_REGION: ${WATSONX_REGION}
       BAM_API_KEY: ${BAM_API_KEY:-dummy}
       OPENAI_API_KEY: ${OPENAI_API_KEY:-dummy}
+      OLLAMA_HOST: ${OLLAMA_HOST:-dummy}
```

Comment:
OLLAMA_URL should be enough, please remove the HOST and MODEL variables, see [bee-api env example](https://github.com/i-am-bee/bee-api/blob/main/.env.example#L53)

Reply:
Have made above change, will test. 
Can squash the changes if preferred (kept distinct during review)

Reply:
with this change (and bee-api at e7ced3f1ce87e8cd621d218531ef8ee961149d39 ), ollama will not work with the sample agent, with the api throwing the original error `fetch failed: connect ECONNREFUSED 127.0.0.1:11434`

Updating bee-api to the latest version ( aba5f3a6e9f6df6f1b10f5119e33a9e8646ae48f ) made no difference

Reverting to the original version in the PR - c38b32b8a5615ba830f46cde6528951186dd2620 and the agent is back working again.

Currently both OLLAMA_HOST and OLLAMA_URL are needed to work correctly

For clarity here's one of the failures if this is not done:
```
{"level":"error","time":"2024-11-01T09:53:22.866Z","hostname":"1bf647cc52cb","name":"bee-api","queueName":"runs","job":{"id":"run_6724a5123ac743ab9b429566","name":"runs","failedReason":"fetch failed","data":{"runId":"run_6724a5123ac743ab9b429566"}},"err":{"type":"TypeError","message":"fetch failed: connect ECONNREFUSED 127.0.0.1:11434","stack":"TypeError: fetch failed\n    at node:internal/deps/undici/undici:12502:13\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async post (file:///app/node_modules/.pnpm/ollama@0.5.9/node_modules/ollama/dist/shared/ollama.133b951a.mjs:112:20)\n    at async Ollama.show (file:///app/node_modules/.pnpm/ollama@0.5.9/node_modules/ollama/dist/shared/ollama.133b951a.mjs:383:22)\n    at OllamaChatLLM.meta (/app/node_modules/.pnpm/bee-agent-framework@0.0.33_@bufbuild+protobuf@1.10.0_@googleapis+customsearch@3.2.0_@grpc+grp_rcgixnyepg3zlv6xeprvpr6bga/node_modules/bee-agent-framework/src/adapters/ollama/chat.ts:147:19)\n    at TokenMemory.add (/app/node_modules/.pnpm/bee-agent-framework@0.0.33_@bufbuild+protobuf@1.10.0_@googleapis+customsearch@3.2.0_@grpc+grp_rcgixnyepg3zlv
6xeprvpr6bga/node_modules/bee-agent-framework/src/memory/tokenMemory.ts:84:20)\n    at TokenMemory.addMany (/app/node_modules/.pnpm/bee-agent-framework@0.0.33_@bufbuild+protobuf@1.10.0_@googleapis+customsearch@3.2.0_@grpc+grp_rcgixnyepg3zlv6xeprvpr6bga/node_modules/bee-agent-framework/src/memory/base.ts:39:13)\n    at async executeRun (file:///app/dist/runs/execution/execute.js:75:3)\n    at async file:///app/dist/runs/jobs/runs.queue.js:49:7\n    at Worker.processJob (/app/node_modules/.pnpm/bullmq@5.8.1/node_modules/bullmq/src/classes/worker.ts:786:22)\ncaused by: Error: connect ECONNREFUSED 127.0.0.1:11434\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)\n    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)"},"msg":"Job failed"}
```

As above, I've left the explicit revert in the history - will squash when all is good with the fix...

Reply:
* bee-api [only uses **OLLAMA_URL** ](https://github.com/search?q=org%3Ai-am-bee+OLLAMA_URL&type=code)
* bee-agent-framework uses [OLLAMA_HOST](https://github.com/search?q=org%3Ai-am-bee%20OLLAMA_HOST&type=code)

An alternative fix would be to consolidate these settings into one, and update all references? 
I proposed a compose fix only here as it had the least impact